Modules/binascii.c

```diff
@@ -592,6 +692,511 @@ binascii_b2a_base64_impl(PyObject *module, Py_buffer *data, int newline)
     return PyBytesWriter_FinishWithPointer(writer, ascii_data);
 }
 
+/*[clinic input]
+binascii.a2b_ascii85
+
+    data: ascii_buffer
+    /
+    *
+    fold_spaces: bool = False
+        Allow 'y' as a short form encoding four spaces.
+    wrap: bool = False
+        Expect data to be wrapped in '<~' and '~>' as in Adobe Ascii85.
+    ignore: Py_buffer(c_default="NULL", py_default="b''") = None
+        An optional bytes-like object with input characters to be ignored.
```

Comment:
I have a question -- why do you use different parameter names than in `base64.a85encode()`?

Reply:
I just chose some names that sounded OK to me. I didn't like `adobe` in `base64.a85decode()` because it seemed too specific, and there wasn't commonality between `binascii` and `base64` parameter names (lots of legacy code everywhere).

After reading #143262 and #143216 I gather that reusing CPython interned strings is implicitly good practice, so I will try to do that if you prefer.